cpp/tests/strings/chars_types_tests.cpp

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  */
 
-#include <tests/strings/utilities.h>
 #include <cudf/column/column.hpp>
 #include <cudf/strings/char_types/char_types.hpp>
+#include <cudf/strings/convert/convert_floats.hpp>
+#include <cudf/strings/convert/convert_integers.hpp>
```

Comment:
The gtests for `cudf::strings::is_integers()` and `cudf::strings::is_floats()` should be moved to `integers_tests.cpp` and `floats_tests.cpp` appropriately as well.